donkeycar/pipeline/types.py

```diff
@@ -70,22 +71,50 @@ class TubDataset(object):
     '''
 
     def __init__(self, config: Config, tub_paths: List[str],
-                 shuffle: bool = True) -> None:
+                 shuffle: bool = True, select: Optional[str] = None) -> None:
```

Comment:
Same here, This should not be a string, and we should not have to parse it at all.